types-packages/uri.d.ts

```diff
@@ -0,0 +1 @@
+export type IdeUri = string
```

Comment:
I'm not a fan of this approach. `string` is more readable, and if you're thinking "documentation purposes", renaming `string` to `IdeUri` is a very poor substitution for documentation.

Reply:
This is a workaround for the TypeScript limitation. `IdeUri` is not any `string`, but it is a certain subset of strings. Choosing a proper alias name shows this.

This also allows to change `string` to something more accurate later on. 

This is also needed for backward compatibility.